tests/rest_api/test_0100_jobs.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
```

Comment:
The name of this file led me to think why do we use numbers in filenames? If these numbers are not needed then I suggest to follow pattern `test_{resource_name}.py` or if we have some reasons to keep this numbers in filenames than it's better to describe (in README) a convention of assignment of this numbers.